client-src/elements/form-definition.js

```diff
@@ -389,6 +389,9 @@ const FLAT_PREPARE_TO_SHIP_FIELDS = {
         'webview_risks',
         'anticipated_spec_changes',
         'i2s_lgtms',
+        'availability_expectation',
+        'adoption_expectation',
+        'adoption_plan',
```

Comment:
Are these fields intended to be available and visible for every feature?

Reply:
I think just for every feature with feature_type of "new feature incubation".